### ISSUE TITLE
Add possibility to use row.names or row numbers as firm ID

### ISSUE THREAD

**Issue by ohjakobsen** [Riksrevisjonen/pioneeR#42]
If the user uploads a data set without firm IDs, the app assumes that we have time series data if there are duplicate values on the variable selected as the ID (usually the first column in the data set). This can lead to unexpected behavior. The user should be able to select row names or row numbers (if no names exists) as the firm ID. See also #41.

The `hospitals` data set in the rDEA package is a good example of this issue.

```r
runPioneeR(x = rDEA::hospitals)
```

**Comment by ohjakobsen** [comment]
Not a problem after #53. Open new issue if needed, we'll close this for now.
